core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/ManagementPackV4Controller.java

```diff
@@ -60,4 +62,11 @@ public ManagementPackV4Response deleteInWorkspace(Long workspaceId, String name)
         notify(ResourceEvent.MANAGEMENT_PACK_DELETED);
         return converterUtil.convert(deleted, ManagementPackV4Response.class);
     }
+
+    @Override
+    public ManagementPackV4Responses deleteMultipleInWorkspace(Long workspaceId, Set<String> names) {
+        Set<ManagementPack> deleted = mpackService.deleteMultipleByNameFromWorkspace(names, workspaceId);
+        notify(ResourceEvent.MANAGEMENT_PACK_DELETED);
```

Comment:
I think we will send false information in some case. 
Do we delete all or nothing? Can we delete a particular set of _names_?
I mean, we get 3 names, _name1, name2, name3_. We delete successfully name1, name2, but name3 failed. Do we send _success_ in this case?